tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
 skipsdist=True
-envlist=py36-dj111-wag{113,23},
-        py36-dj20-wag23,
+envlist=py36-dj{111,20}-wag23,
         py{36,38}-dj{20,22}-wag28,
-        flake8
+        lint
```

Comment:
Could the `lint` env appear first in this list, so that if it fails, the entire run fails quickly?

Reply:
Added `lint` to start of list